### PR TITLE
week4-TDD

### PR DESCRIPTION
**Pull request by marmot0814**
Implement query function and pass all tests

### CODE REVIEW

File: walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 1]
```diff
@@ -15,9 +15,64 @@
...
+
+      boolean hasRelatedPerson = false;
+      for (String attendee : event.getAttendees())
+        if (attendees.contains(attendee))
+          hasRelatedPerson = true;
+      
+      if (!hasRelatedPerson)
+        continue;
+
+      if (timeSlot.get(when.start()) == null)
```
Comment (louisc1028) [inline walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:43]: Could you help add comments and explains what the timeSlot does and how it stored the time info?

Comment (marmot0814) [reply walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:43]: OK, I will write it in the code soon.

Comment (marmot0814) [reply walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:71]: Added!

Comment (louisc1028) [reply walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:71]: Thanks for the well-explained comments!

**Pull request merged**